config/routes.rb

```diff
@@ -23,10 +23,13 @@
   # Field type path
   resources :field_types, only: %i(show create update)
 
+  # User path
   resources :users
 
   namespace :admin do
     resources :users, only: :index
   end
+  # Booking history path
+  get "bookings/history", to: "bookings#booking_history", as: "booking_history"
```

Comment:
```suggestion
  get "bookings/history", to: "bookings#index", as: "booking_history"
```

chị nghĩ là ko cần phải tự defind 1 name mới đâu em, nếu em làm web thì tạm tha, nhưng nếu là API và em là 1 service trong hệ sinh thái service thì rõ ràng chỉ có mỗi em chơi dc thôi

ở đây em có thể chọn 1 trong 2 cách bên dưới
1. sử dụng url `booking/` -> gọi đến bookings#index
2. Hoặc có thể chọn theo hướng booking là phải thuộc user nào ( thường là url bên ADMIN chứ urser thì hiển thị user_id nó ko có ý nghĩa lắm) sẽ có style: `users/:user_id/bookings` => cụ thể thì nó cũng call đến bookings#index thôi, chỉ là trong file routes thì sẽ dạng nested resource ấy  (nested resource là gì thì tự nhớ lại, gõ miết mỏi tay)